### PR TITLE
Add license information display when clicking the Restriction icon

### PR DESCRIPTION
**Pull request by riyenas0925**
## Description
* Add license information display when clicking the Restriction icon

## Type of change
Please insert 'x' one of the type of change.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by soimkim** [comment]
@riyenas0925 ,
In my opinion, it needs indentation.

**Comment by riyenas0925** [comment]
@soimkim 
The indentation has been changed :smile:

If you check the code, the function below is currently used repeatedly. the function of src_fn > showLicenseView is duplicated, so it seems difficult to maintain.
Is there a common jsp file to contain these functions?

```js
var src_fn = {
        // License 상세 페이지 이동
        showLicenseViewPage : function(gridNm, rowid){
        
        ...중략
        
        }
}
```

If you tell us a file where we can manage these functions in common, we will refactor the current code to eliminate duplicates.

File: src/main/webapp/WEB-INF/views/admin/partner/edit-js.jsp [hunk 1]
```diff
@@ -2022,5 +2022,32 @@ var sampleFile =  ${ct:getAllValuesJson(ct:getConstDef('CD_SAMPLE_FILE'))};
 			}
 		}
 	}
+
+	var src_fn = {
+        // License 상세 페이지 이동
+        showLicenseViewPage : function(gridNm, rowid){
+            console.log("test : " + gridNm);
```
Comment (astrod) [inline src/main/webapp/WEB-INF/views/admin/partner/edit-js.jsp:2029]: It would be a good idea to remove console.log when commiting

Comment (riyenas0925) [reply src/main/webapp/WEB-INF/views/admin/partner/edit-js.jsp:2029]: Thanks for your comments.
Pull Request still in progress, so I'll reflect it when it's complete!

Comment (riyenas0925) [reply src/main/webapp/WEB-INF/views/admin/partner/edit-js.jsp]: @astrod 
I've reflected your opinion Thank you for commenting!

**Review by soimkim — APPROVED** [review]

**Pull request merged**